compose/scripts/publishExtendedIcons

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+cd "$(dirname "$0")"
+. ./prepare
+
+pushd ..
+./gradlew publishComposeJbExtendedIcons $COMPOSE_DEFAULT_GRADLE_ARGS --max-workers=1 "$@" || exit 1
```

Comment:
Why separate script just for that?

Reply:
- it is very heavy task (on CI: 20min - publishComposeJbExtendedIcons, 6min - publish).
- it shouldn't be run in parallel (it requires a lot of RAM)
- Compose can work without extended icons. So on the local machine we don't have to wait 20min to build icons which we don't use in the local project

Reply:
Thanks, makes sense.